cfme/tests/infrastructure/test_pxe_provisioning.py

```diff
@@ -122,4 +122,4 @@ def test_pxe_provision_from_template(appliance, provider, vm_name, smtp_test, se
             'vlan': pxe_vlan}}
 
     do_vm_provisioning(appliance, pxe_template, provider, vm_name, provisioning_data, request,
-                       smtp_test, num_sec=2100)
+                       smtp_test, num_sec=3600)
```

Comment:
post provision script takes extremely long time to finish. not sure we need that stuff as it's not related to this tests in particular